qupath_plus.def

```diff
@@ -0,0 +1,54 @@
+Bootstrap: docker
+From: amd64/ubuntu:jammy
+
+%runscript
+    cd /qupath/QuPath/bin
+    ./QuPath
+
+%post
+# install needed dependencies
+    rm -rf /var/lib/apt/lists/*
+    apt-get update -y
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+         ca-certificates \
+         wget \
+         libgl1 \
+         xz-utils \
+         openjfx \
+         unzip
```

Comment:
This was needed for unzipping the zip files, seems like by default only gunzip was in the container.